backend/lib/endpoints/posts.js

```diff
@@ -113,6 +112,22 @@ async function routes(app) {
         filters.push({ types: { $in: type } });
       }
 
+      //workMode filter
+      if (workMode){
+        const workModes = workMode.map(mode => mode.toLowerCase())
+        if(workModes.includes('both')){
+          console.log("In both")
+          filters.push({ $or: [ { $and: [ { workMode: null }, {types: { $nin : ['Remote Work'] } } ] }, {workMode: { $in: workModes } }] });
+        }
+        else if(workModes.includes('remote')) {
+          console.log("In remote")
+          filters.push({ $or: [ { $and: [ { workMode: null }, {types: { $in : ['Remote Work'] } } ] }, {workMode: { $in: workModes } }] });
```

Comment:
This query seems complex, might be a better (simpler) way to achieve the goal ❔ 
it seems you are not using formatting tool Aditi, the format changes when I run prettier in VS code

Reply:
@ddveloper this one is basically adds logic to filter old records which don't have workMode fields but can have remote work tag in types feild.